lib/libutils/isoc/bget_malloc.c

```diff
@@ -609,7 +613,11 @@ void *mdbg_malloc(const char *fname, int lineno, size_t size)
 {
 	struct mdbg_hdr *hdr;
 
+#if defined(ARM64)
+	COMPILE_TIME_ASSERT(sizeof(struct mdbg_hdr) == sizeof(uint32_t) * 6);
```

Comment:
After checking *Procedure Call Standard for the ARM 64-bit Architecture (AArch64)* it looks like it should be 16 for ARM64.